### ISSUE TITLE
What should I do if the upgrade fails more than ten times? Ask for help

### ISSUE THREAD

**Issue by golden1688** [PrestaShop/PrestaShop#21750]
Hello there!
Strong technical support:
I am a user of prestashop, I upgraded from prestashop 1.7.6.1 to 1.7.6 stable version (1.7.6.8), I tried to upgrade several times, and it appeared: [Ajax/Server operation backup file error] textStatus: "Error" errorThrown: "" JqXHR: "", what should I do? Please help me, thanks!
PHP version is 7.2

**Comment by hibatallahAouadni** [comment]
Hello @golden1688 ,

Thanks for your issue. Unfortunately, there is not enough information provided to reproduce it and work on it. **You must follow the template to submit a complete bug report**.

Read more about how we expect the issues to be handled [here](https://build.prestashop.com/news/how-bug-reports-are-handled/). Feel free to create a new issue if you can provide further information, we will be glad to investigate it

Thanks!

**Comment by matks** [comment]
@golden1688 you should have some more informations inside the logs in `/{Prestashop_Folder}/{Admin_folder}/autoupgrade/tmp/`

**Comment by golden1688** [comment]
> 你好@ golden1688，
> 
> 感谢您的问题。不幸的是，没有提供足够的信息来进行复制和处理。**您必须按照模板提交完整的错误报告**。
> 
> [在此处](https://build.prestashop.com/news/how-bug-reports-are-handled/)阅读有关我们期望如何处理这些问题的更多信息。如果您可以提供更多信息，请随时提出新问题，我们将很乐意对此进行调查
> 
> 谢谢！

Hello there! HibatallahAouadni!
There are a lot of server logs, which are security information. Can I send you a private message?

**Comment by hibatallahAouadni** [comment]
Hello @golden1688 

As discussed with @khouloudbelguith, in the 1-Click Upgrade configuration page, you should disable **Back up my images**, because the Upgrade was stopped during the backup. (see the screenshot below)

![image](https://user-images.githubusercontent.com/23308427/98256565-fe6f9d00-1f7e-11eb-9118-4b9c04fb04bf.png)

Thanks!

**Comment by golden1688** [comment]
Hello there!
Great technical support:
The picture backup you described has been set up.
I repeatedly looked at the questions and answers on the forum, but did not find a good way. I communicated with the server a lot and adjusted many PHP parameters.
max_file_uploads from 50 => 50
memory_limit from 1 G => 2G
post_max_size from 256M=> 1G
The upgrade is still backed up until there are 232 data tables left, and it doesn't continue. What should I do? I have thought of a lot of ways, can you still find a way? Please help! thank!
                                                                  Your loyal customer Kim
![Uploading 001.jpeg…]()
![Uploading 002.jpeg…]()
![Uploading 003.jpeg…]()

**Comment by golden1688** [comment]
Hello there!
Great technical support:
I really can’t help it. There are 232 forms left during the upgrade, but I still don’t move. I know that the time to connect to the server is up, and it’s disconnected. I have posted the upgrade time for three weeks. I went to the forum to find the answer. , Did not find, what should I do? thank!
                                                 Your loyal customer Kim
![Uploading 3333.jpeg…]()
